docs/datatypes/chain.md

```diff
@@ -100,48 +110,40 @@ def fromSeq[A](s: Seq[A]): Chain[A] =
-
-```scala mdoc
-import cats.data._
-
-NonEmptyChain(1, 2, 3, 4)
-
-NonEmptyChain.fromNonEmptyList(NonEmptyList(1, List(2, 3)))
-NonEmptyChain.fromNonEmptyVector(NonEmptyVector(1, Vector(2, 3)))
+To get a good idea of performance of `Chain`, here are some benchmarks that test monoidal append (higher score is better):
 
-NonEmptyChain.one(1)
+```
+[info] Benchmark                                  Mode  Cnt   Score   Error  Units
+[info] CollectionMonoidBench.accumulateChain     thrpt   20  51.911 ± 7.453  ops/s
+[info] CollectionMonoidBench.accumulateList      thrpt   20   6.973 ± 0.781  ops/s
+[info] CollectionMonoidBench.accumulateVector    thrpt   20   6.304 ± 0.129  ops/s
 ```
 
+As you can see accumulating things with `Chain` is more than 7 times faster than `List` and over 8 times faster than `Vector`.
+So appending is a lot more performant than the standard library collections, but what about operations like `map` or `fold`?
+Fortunately we've also benchmarked these (again, higher score is better):
 
-
-You can also create an `Option` of `NonEmptyChain` from a `Chain` or any other collection type:
-
-```scala mdoc
-import cats.data._
-
-NonEmptyChain.fromChain(Chain(1, 2, 3))
-NonEmptyChain.fromSeq(List.empty[Int])
-NonEmptyChain.fromSeq(Vector(1, 2, 3))
+```
+[info] Benchmark                           Mode  Cnt          Score         Error  Units
+[info] ChainBench.foldLeftLargeChain      thrpt   20        117.267 ±       1.815  ops/s
+[info] ChainBench.foldLeftLargeList       thrpt   20        135.954 ±       3.340  ops/s
+[info] ChainBench.foldLeftLargeVector     thrpt   20         61.613 ±       1.326  ops/s
+[info]
+[info] ChainBench.mapLargeChain           thrpt   20         59.379 ±       0.866  ops/s
+[info] ChainBench.mapLargeList            thrpt   20         66.729 ±       7.165  ops/s
+[info] ChainBench.mapLargeVector          thrpt   20         61.374 ±       2.004  ops/s
 ```
 
-Sometimes, you'll want to prepend or append a single element to a chain and return the result as a `NonEmptyChain`:
+While not as dominant, `Chain` holds its ground fairly well.
+It won't have the random access performance of something like `Vector`, but in a lot of other cases, `Chain` seems to outperform it quite handily.
+So if you don't perform a lot of random access on your data structure, then you should be fine using `Chain` extensively instead.
 
-```scala mdoc
-import cats.data._
+So next time you write any code that uses `List` or `Vector` as a `Monoid`, be sure to use `Chain` instead!
+You can also check out the benchmarks [here](https://github.com/typelevel/cats/blob/v1.3.0/bench/src/main/scala/cats/bench).
 
-NonEmptyChain.fromChainAppend(Chain(1, 2, 3), 4)
-NonEmptyChain.fromChainAppend(Chain.empty[Int], 1)
-NonEmptyChain.fromChainPrepend(1, Chain(2, 3))
-```
+[nec]: @API_LINK_BASE@/cats/data/index.html#NonEmptyChain:cats.data.NonEmptyChainImpl.type
```

Comment:
This is what the mdoc var is for. Laika can't link to this, but this seemed like a reasonable approximation of linking without hardcoding too much.

Reply:
Laika's own ${vars} also don't work in links, I tried that first 🤷‍♂️

Reply:
> Laika can't link to this

If you have a moment, might be good to open an issue for this in Laika.